hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.cpp

```diff
@@ -1137,9 +1036,8 @@ namespace hazelcast {
                     return msg;
                 }
 
-                ClientMessage map_forceunlock_encode(const std::string &name, const serialization::pimpl::data &key,
-                                                     int64_t reference_id) {
-                    size_t initial_frame_size = ClientMessage::REQUEST_HEADER_LEN + ClientMessage::INT64_SIZE;
+                ClientMessage map_forceunlock_encode(const std::string &name, const serialization::pimpl::data &key, int64_t reference_id) {
+                    size_t initial_frame_size = ClientMessage::REQUEST_HEADER_LEN  + ClientMessage::INT64_SIZE;
```

Comment:
There are 2 space characters before the `+`

Reply:
These existed in the protocol repo and was deleted manually here. They came back since I regenerated. I fixed this in the protocol repo (see hazelcast/hazelcast-client-protocol@3a0bf96) and now it's fine.